eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/model/dto/CommentDTO.java

```diff
@@ -0,0 +1,44 @@
+package hu.flowacademy.companycalendar.model.dto;
+
+import hu.flowacademy.companycalendar.model.Comment;
+import hu.flowacademy.companycalendar.model.Meeting;
+import hu.flowacademy.companycalendar.model.User;
+import hu.flowacademy.companycalendar.repository.MeetingRepository;
+import hu.flowacademy.companycalendar.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class CommentDTO {
+
+    private Long id;
+    private String content;
+    private Long userId;
+    private Long meetingId;
+
+    public CommentDTO(Comment comment) {
+        this.id = comment.getId();
+        this.content = comment.getContent();
+        if (comment.getUser() != null) {
+            this.userId = comment.getUser().getId();
+        } else {
+            throw new RuntimeException("User not found with this id");
```

Comment:
Ezt a null checket tegyük a konstruktor elejére, így azonnal elszáll, és nem állítgat semmit feleslegesen. Pluszban hozzácsaphatjuk a 37. sorban a másik RuntimeException dobást is. Azok helyett dobhatnánk inkább egy `IllegalArgumentException`-t, az talán jobban illik ide.